jenkins.py

```diff
@@ -324,7 +324,7 @@ def read_and_post_job_metrics(module_config, url, job_name, last_timestamp):
     resp_obj = get_response(job_url, 'jenkins', module_config)
     extra_dimensions = {}
     extra_dimensions['Job'] = job_name
-    if resp_obj and resp_obj['builds']:
+    if resp_obj is not None and "builds" in resp_obj and resp_obj['builds']:
```

Comment:
I would consider changing this line to this:
`if isinstance(resp_obj, dict) and resp_obj.get('builds', None): `
This way we know for sure the resp_obj is a dictionary and not None or something else.  It also simplifies the checks for 'builds'.